cpp/daal/src/algorithms/svm/svm_predict_impl.i

```diff
@@ -192,9 +192,10 @@ struct SVMPredictImpl<defaultDense, algorithmFPType, cpu> : public Kernel
         ReadColumns<algorithmFPType, cpu> mtSVCoeff(*svCoeffTable, 0, 0, nSV);
         DAAL_CHECK_BLOCK_STATUS(mtSVCoeff);
         const algorithmFPType * const svCoeff = mtSVCoeff.get();
+        const size_t nOptimalSizeBlock        = nSV > 2048 ? 256 : 2048;
```

Comment:
Probably, we need to use cache sizes to define optimal block size.
But not in this PR, of course.

Reply:
Yes it will be right. It is necessary to make experiments in the future.